--- qtwebkit-2.3.3/Source/ThirdParty/ANGLE/src/compiler/glslang.y.omv~	2014-08-09 09:09:27.797691521 +0000
+++ qtwebkit-2.3.3/Source/ThirdParty/ANGLE/src/compiler/glslang.y	2014-08-09 09:10:16.643252465 +0000
@@ -47,6 +47,7 @@ WHICH GENERATES THE GLSL ES PARSER (glsl
 %expect 1 /* One shift reduce conflict because of if | else */
 %pure-parser
 %parse-param {TParseContext* context}
+%lex-param {YYLEX_PARAM}
 
 %union {
     struct {
